credentials/apps/edx_credentials_extensions/edly_credentials_app/api/tests/test_serializers.py

```diff
@@ -0,0 +1,85 @@
+"""
+Tests for edly_api serializers.
+"""
+from django.test import TestCase
+
+from credentials.apps.catalog.tests.factories import ProgramFactory
+from credentials.apps.core.tests.mixins import SiteMixin
+from credentials.apps.credentials.tests.factories import SignatoryFactory, ProgramCertificateFactory
+from credentials.apps.edx_credentials_extensions.edly_credentials_app.api.serializers import SignatorySerializer, ProgramCertificateConfigurationSerializer
+
+
+class SignatorySerializerTest(SiteMixin, TestCase):
+    """
+    Tests for Signatory Serializer.
+    """
+
+    def setUp(self):
+        super(SignatorySerializerTest, self).setUp()
+        self.serializer = SignatorySerializer
+        self.test_data = {
+            "name": "Jhon Doe",
```

Comment:
Be consistent with using single quotes in code and use double quotes when necessary or while pointing to names (in docstrings and code comments).

Reply:
Done